ENVIRONMENT.rst

```diff
@@ -29,3 +29,8 @@ Environment Configuration Settings
 - **USE_WALE**: whether to consider base backup/restore with WAL-E.
 - **GOOGLE_APPLICATION_CREDENTIALS**: credentials for WAL-E when running in Google Cloud.
 - **CALLBACK_SCRIPT**: the callback script to run on various cluster actions (on start, on stop, on restart, on role change). The script will receive the cluster name, connection string and the current action. See `Patroni <http://patroni.readthedocs.io/en/latest/SETTINGS.html?highlight=callback#postgresql>`__ documentation for details.
+- **LOG_S3_BUCKET**: path to the S3 bucket used for PostgreSQL daily log files (i.e. s3://foobar). Spilo will add /spilo/scope/pg_daily_logs to that path.
+- **SHIP_LOG_TO_S3**: whether to ship compressed logs from ``pg_log`` to S3 bucket. Disabled by default.
```

Comment:
Redundant. If the bucket and schedule are defined that already means we want to push logs.